src/deroff.rs

```diff
@@ -1,8 +1,40 @@
 /// A translation of https://github.com/fish-shell/fish-shell/blob/e7bfd1d71ca54df726a4f1ea14bd6b0957b75752/share/tools/deroff.py
 // """ Deroff.py, ported to Python from the venerable deroff.c """
+use std::collections::HashMap;
+
+type TODO_TYPE = u8;
```

Comment:
Types should be `UpperCamelCase` as per https://rust-lang-nursery.github.io/api-guidelines/naming.html

Reply:
In general, I agree. I think as a placeholder/todo type declaration the screaming snake case is probably fine.